docs/nfl.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en-US" xmlns="http://www.w3.org/1999/xhtml" xmlns:og="http://ogp.me/ns#" xmlns:fb="http://www.facebook.com/2008/fbml">
+
+<head>
+    <meta charset="UTF-8">
+    <title>MakeCode Arcade and NFL</title>
+    <meta name="Description" content="Football in MakeCode Arcade" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
+
+    <link rel="stylesheet" data-rtl="/blb/rtlsemantic.css" href="/doccdn/semantic.css">
+
+    <script src="/doccdn/jquery.js" type="text/javascript"></script>
```

Comment:
where is this being used?

Reply:
Was being used for for semantic.js at one point, but now that i look at it that isn't used anymore here, removed~